package.json

```diff
@@ -90,8 +92,10 @@
     "express": "^4.17.1",
     "fetch-sparql-endpoint": "^1.8.0",
     "handlebars": "^4.7.6",
+    "koa": "^2.13.0",
```

Comment:
Couple of questions here:

- How come CSS itself has to depend on Koa and not just `oidc-provider`?
- What is our exact relationship to and dependency on Koa?
  - Because at the moment, we only have a weak dependency on Express that could actually be removed (#370).
  - We could also replace Express by Koa, depending on the answers heree.

Reply:
The main reason for including it as a dependency is so we can do additional routing. It's just easier to use koa and express than saying `if (req.url === '/something') {}`. The IdP requires more traditional routing than the LdP

Reply:
Or we can configure that in the dependency injection and not write that code at all 🙂 Continued below.